src/DI/HTTPServices.php

```diff
@@ -0,0 +1,135 @@
+<?php
+/* Copyright (c) 2016 Richard Klees <richard.klees@concepts-and-training.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\DI;
+
+use ILIAS\HTTP\Cookies\CookieJarFactory;
+use ILIAS\HTTP\GlobalHttpState;
+use ILIAS\HTTP\Request\RequestFactory;
+use ILIAS\HTTP\Response\ResponseFactory;
+use ILIAS\HTTP\Response\Sender\ResponseSenderStrategy;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+
+/**
+ * Provides an interface to the ILIAS HTTP services.
+ *
+ * @author  Nicolas Schäfli <ns@studer-raimann.ch>
+ */
+class HTTPServices implements GlobalHttpState {
+
+	/**
+	 * @var ResponseSenderStrategy
+	 */
+	private $sender;
+
+    /**
+     * @var CookieJarFactory $cookieJarFactory
+     */
+	private $cookieJarFactory;
+
+    /**
+     * @var RequestFactory $requestFactory
+     */
+	private $requestFactory;
+
+    /**
+     * @var ResponseFactory $responseFactory
+     */
+	private $responseFactory;
```

Comment:
Please use tabs for indentation. I'm would prefer 4 spaces as well, but [coding guidelines](http://www.ilias.de/docu/ilias.php?ref_id=42&from_page=29964&obj_id=202&cmd=layout&cmdClass=illmpresentationgui&cmdNode=fr&baseClass=ilLMPresentationGUI)...

You have spaces in some other files as well.